### PR TITLE
Add Looka card to the Marketing Tools page

### PR DESCRIPTION
**Pull request by belcherj**
#### Changes proposed in this Pull Request

* Adds Looka Marketing Tools card

#### Testing instructions

* Go to Marketing tools page an observe Looka card.
* Make sure it clicks out to Looka.

### CODE REVIEW

**Comment by jancavan** [comment]
Link looks good to me.

For the copy:

`A custom logo helps you brand stand out and makes your memorable. Our partners Logojoy and Looka are standing by if you’d like some professional help.`

We should omit Logojoy, they just rebranded as Looka. So it should be:

`A custom logo helps you brand stand out and makes your memorable. Our partner Logojoy is standing by if you’d like some professional help.`

@michelleweber Is the above sentence correct or should it be:

`A custom logo helps you brand stand out and makes your memorable. Our partner, Logojoy, is standing by if you’d like some professional help.`

With commas before and after Logojoy?

**Comment by michelleweber** [comment]
I'd either go with the no-comma version here. There are also a few typos in the above, and on a second read I want to avoid using "stand" and "standing," so let's change to:

A custom logo helps your brand pop and makes your site memorable. Our partner Logojoy is standing by if you’d like some professional help.

**Comment by huguesvincent** [comment]
Let's remove all mention of Logojoy and replace it with Looka. As of this morning, all the URL will be swapped to the new branding. I'll update the support page as well.

**Comment by belcherj** [comment]
Copy updated to: `A custom logo helps your brand pop and makes your site memorable. Our partner Looka is standing by if you’d like some professional help.`

File: client/my-sites/marketing/tools/index.tsx [hunk 1]
```diff
@@ -63,6 +63,17 @@ export const MarketingTools: FunctionComponent< MarketingToolsProps > = ( {
...
 				</MarketingToolsFeature>
+				<MarketingToolsFeature
+					title={ translate( 'Want to build a great brand? Start with a great logo' ) }
+					description={ translate(
+						'A custom logo helps your brand pop and makes your site memorable. Our partner Looka is standing by if you’d like some professional help.'
+					) }
+					imagePath="/calypso/images/illustrations/branding.svg"
+				>
+					<Button href={ 'https://looka.com/wordpress-users/' } target="_blank">
+						{ translate( 'Create a logo' ) }
```
Comment (stephanethomas) [inline client/my-sites/marketing/tools/index.tsx]: We should uppercase the first letter of each word.

**Comment by jancavan** [comment]
Yes, I meant Looka, sorry my bad! Thank you!

**Pull request merged**